src/rendering/webgl/programs/common/program.ts

```diff
@@ -70,21 +79,28 @@ export abstract class Program<Uniform extends string = string> implements Abstra
   capacity = 0;
   verticesCount = 0;
 
-  abstract getDefinition(): ProgramDefinition<Uniform>;
+  isInstanced: boolean;
+  constantBuffer: WebGLBuffer = {} as WebGLBuffer;
```

Comment:
Is it normal to set a default buffer as an empty JavaScript object? Just a trick to avoid nastiness of nullable property before constructor is over?

Reply:
Yes, just that trick...